app/src/main/java/com/google/samples/apps/nowinandroid/navigation/NiaNavHost.kt

```diff
@@ -39,33 +39,32 @@ import com.google.samples.apps.nowinandroid.ui.NiaAppState
 @Composable
 fun NiaNavHost(
     appState: NiaAppState,
+    snackbarHostState: SnackbarHostState,
     modifier: Modifier = Modifier,
     startDestination: String = forYouNavigationRoute,
 ) {
     val navController = appState.navController
+    val interestsScrollState = rememberLazyListState()
```

Comment:
Having this here is a little strange, maybe this is related to the crash?

Reply:
Hey Alex, it was a weird crash. I don't think this line is related. The crash appeared after moving the NiaTopBar to the Scaffold's topBar slot.